azerothcore/azerothcore-wotlk
# fix (Core/Raid/Boss Script): Naxxramas boss loatheb kill event

###### ISSUES ADDRESSED:
When boss loatheb it's killed in Naxxramas, players remains  trapped, and instance event is still running.

Closes 
https://github.com/azerothcore/azerothcore-wotlk/issues/2053

##### TESTS PERFORMED:
 Build without errors and tested in-game. (Win10 and Debian 9 servers).

##### HOW TO TEST THE CHANGES:
- Enter instance/raid Naxxramas 
- Kill boss loatheb
- Door should open and also teleport portal  be active as intended 

##### Target branch(es):

Master
